### PR TITLE
GUI: Add "use original gui" checkbox in Loom and MI CD

### PR DESCRIPTION
**Pull request by tag2015**
Add the "use original gui" checkbox to the custom Loom EGA/VGA and Monkey 1 CD version game options window.
Also slightly change to layout for consistency with the other games.

### CODE REVIEW

**Comment by tag2015** [comment]
Regarding the layout change, the sliders are simply moved to the bottom after the checkboxes
cbfix

**Comment by AndywinXp** [comment]
Perfetto, grazie! If this gets the approval of eriktorbjorn or anyone else who had the chance of working on top of the GUI code, we should be all set.

**Comment by eriktorbjorn** [comment]
I think the only reason I put the sliders above the checkboxes was that I thought they looked more like the widgets above. I can't say I care much either way.

I was thinking of allowing both dynamic and static settings at the same time, but the simple test hack I made didn't work (I think it was trying to squeeze both into the same space, causing them to overlap), and I haven't had the time to look any further since then.

**Comment by tag2015** [comment]
> I think the only reason I put the sliders above the checkboxes was that I thought they looked more like the widgets above. I can't say I care much either way.
Well I moved the sliders to the bottom simply because the other games have checkboxes too, and since the sliders are an addition only for loom/mi cd they seemed more appropriate at the bottom. However it's a minor thing, the main reason for the PR was to add the original gui checkboxes, since mi cd / loom vga are now supported (and loom ega soon I think)

File: engines/scumm/dialogs.cpp [hunk 1]
```diff
@@ -962,6 +966,8 @@ LoomEgaGameOptionsWidget::LoomEgaGameOptionsWidget(GuiObject *boss, const Common
 	_overtureTicksValue->setFlags(GUI::WIDGET_CLEARBG);
 
 	_enableEnhancementsCheckbox = createEnhancementsCheckbox(widgetsBoss(), "LoomEgaGameOptionsDialog.EnableEnhancements");
+	_enableOriginalGUICheckbox = createOriginalGUICheckbox(widgetsBoss(), "LoomEgaGameOptionsDialog.EnableOriginalGUI");
+	_enableOriginalGUICheckbox->setVisible(false);  // TODO: remove when implemented
```
Comment (bluegr) [inline engines/scumm/dialogs.cpp:970]: Shouldn't this be removed?

Comment (tag2015) [reply engines/scumm/dialogs.cpp:970]: @AndywinXp is currently implementing all the original GUIs in PR [4328](https://github.com/scummvm/scummvm/pull/4328), the Loom EGA is not yet merged, so my idea was to put the control in place but hide it for now, as the checkbox does nothing.

**Comment by sev-** [comment]
Historically, we were keeping the original GUI under things like Ctrl+Shift+F5, but I think it is good to have it mentioned explicitly in the GUI.

I understand, this is a pre-work, so once the main PR lands, this will be take care of.

From the GUI changes perspective, this looks good. I like the consistency with the original in-game interface.

Merging.

**Pull request merged**